packages/core/src/components/Combobox/__stories__/Combobox.stories.tsx

```diff
@@ -237,7 +237,12 @@ export const WithCategories = {
         justify={Flex.justify.START}
         align={Flex.align.START}
       >
-        <StoryDescription description="Regular" vertical align={Flex.align.START}>
+        <StoryDescription
+          description="Regular"
+          vertical
+          // @ts-ignore
```

Comment:
Will resolve it later in storybook-blocks, the components are not typed so it's quite a mess now